bpm/bonita-util/bonita-common-util/src/main/java/org/bonitasoft/engine/bdm/BDMQueryUtil.java

```diff
@@ -58,22 +58,22 @@ public static String createQueryNameForUniqueConstraint(final UniqueConstraint u
         if (uniqueConstraint == null) {
             throw new IllegalArgumentException("uniqueConstraint cannot be null");
         }
-        return getQueryName(uniqueConstraint.getFieldNames());
+        return getQueryName(uniqueConstraint.getFieldNames().toArray(new String[0]));
```

Comment:
The ... notation is more convenient as we often use the method with only one element.